src/overlays/actors/ovl_En_Ge3/z_en_ge3.c

```diff
@@ -30,37 +57,250 @@ const ActorInit En_Ge3_InitVars = {
     (ActorFunc)EnGe3_Update,
     (ActorFunc)EnGe3_Draw,
 };
-*/
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Ge3/func_80A34620.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Ge3/EnGe3_Init.s")
+static ColliderCylinderInit sCylinderInit = {
+    { COLTYPE_UNK10, 0x00, 0x09, 0x39, 0x10, COLSHAPE_CYLINDER },
+    { 0x00, { 0x00000000, 0x00, 0x00 }, { 0x00000722, 0x00, 0x00 }, 0x00, 0x01, 0x01 },
+    { 20, 50, 0, { 0, 0, 0 } },
+};
+
+extern FlexSkeletonHeader D_0600A458; // Same as GeldB
+extern AnimationHeader D_0600B07C; // Idle with right hand on hip and left over mouth
+
+void EnGe3_ChangeAction(EnGe3* this, s32 i) {
+    static EnGe3ActionFunc sActionFunc[] = { EnGe3_WaitLookAtPlayer };
+    static AnimationHeader* sAnimationHeader[] = { 0x0600B07C }; // Idle with right hand on hip and left over mouth
+    static u8 sAnimationMode[] = { ANIMMODE_LOOP };
```

Comment:
Since they are arrays, these should probably be named plural even tho they only have 1 entry in the base game.
And you can go with a simpler `sAnimations` if you want.
```suggestion
    static EnGe3ActionFunc sActionFuncs[] = { EnGe3_WaitLookAtPlayer };
    static AnimationHeader* sAnimations[] = { 0x0600B07C }; // Idle with right hand on hip and left over mouth
    static u8 sAnimationModes[] = { ANIMMODE_LOOP };
```